### PR TITLE
adding 7

### PR DESCRIPTION
**Pull request by SlavaErgonov**
yet another commit!!!

### CODE REVIEW

File: src/test/java/java_Practice/Seventh.java [hunk 1]
```diff
@@ -0,0 +1,7 @@
+package java_Practice;
+
+public class Seventh {
+    public static void main(String[] args) {
+        System.out.println("7");
```
Comment (SlavaErgonov) [inline src/test/java/java_Practice/Seventh.java:5]: What is this???

**Review by AntonSemenov87 — CHANGES_REQUESTED** [review]
fix smith here, idk what

File: src/test/java/java_Practice/Seventh.java (hunk 1 above)
Comment (SlavaErgonov) [reply src/test/java/java_Practice/Seventh.java:5]: Ok

**Review by SlavaErgonov — COMMENTED** [review]
Got it!

**Pull request merged**